### PR TITLE
display results of winner on results page

### PR DESCRIPTION
**Pull request by michaelmld**

### CODE REVIEW

**Comment by a6dang** [comment]
screenshot?

**Comment by michaelmld** [comment]
screen shot 2017-04-03 at 8 30 28 pm

File: controllers/homeController.js [hunk 1]
```diff
@@ -19,15 +19,13 @@ router.post('/', function(req, res) {
             var summonerStats = results[0]
             var summonerStats2 = results[1]
             var commonChampions = findCommonChampions(summonerStats, summonerStats2)
-            console.log(commonChampions);
 
             var compareChampionsResult = compareCommonChampions(commonChampions);
             displayComparisonResults(summonerName, summonerName2, compareChampionsResult);
-            res.render('results', { winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
-
+            res.render('results', { summonerA: summonerName, summonerB: summonerName2, winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
```
Comment (tdashroy) [inline controllers/homeController.js]: Makes sense to me to put this in the displayComparisonResults" function. What do you think?

File: controllers/homeController.js [hunk 2]
```diff
@@ -19,15 +19,13 @@ router.post('/', function(req, res) {
...
         }
         else {
-            console.log("yo i dont work faggot")
+            console.error("failed two get stats for " + summonerName + " or " +summonerName2)
```
Comment (tdashroy) [inline controllers/homeController.js]: two -> to

File: controllers/homeController.js [hunk 3]
```diff
@@ -86,7 +85,7 @@ var compareCommonChampions = function(commonChampions) {
         var championCompare = compareChampionStats(championA.stats, championB.stats);
         compareValue += championCompare.compare;
         return ({
-            id: championA.id,
+            id: championMap[championA.id],
```
Comment (tdashroy) [inline controllers/homeController.js]: is this still an id? If not, we should change the name of the field

File: controllers/homeController.js (hunk 1 above)
Comment (michaelmld) [inline controllers/homeController.js]: To me displayComparisonResults is just there for console logging at the moment and will go away later on, weare starting display results in the view now anyways.

Comment (tdashroy) [inline controllers/homeController.js]: Okay. The intent of that function was to display the results in whatever fashion we would be doing so. But if it's just going to be a single call to render, then not having a separate function for it is fine.

**Pull request merged**